features/product/viewing_products/viewing_product_from_parent_level_taxon_sorted_by_position.feature

```diff
@@ -0,0 +1,28 @@
+@viewing_products
+Feature: Viewing products from specific taxon sorted by position
+    In order to see product in order chosen by shop owner
+    As a Visitor
+    I want to see sorted products
+
+    Background:
+        And the store operates on a channel named "Poland"
+        And the store classifies its products as "Toys"
+        And the "Toys" taxon has child taxon "Soft Toys"
+        And the store has a product "Old pug"
+        And this product is in "Toys" taxon at 1st position
+        And this product is in "Soft Toys" taxon at 3rd position
+        And the store has a product "Small pug"
+        And this product is in "Toys" taxon at 3rd position
+        And this product is in "Soft Toys" taxon at 1st position
+        And the store has a product "Young pug"
+        And this product is in "Toys" taxon at 2nd position
+        And this product is in "Soft Toys" taxon at 2nd position
+
+    @ui
```

Comment:
Can we test it on API? It's shop API so it should be covered IMO.

Reply:
Yes, I can confirm We've got it in API 👌🏽

Reply:
in other PR